src/main/java/core/basesyntax/AreaDrawing.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaDrawing {
```

Comment:
```suggestion
public interface Drawing {
```
краще не прив'язуватись до площі, адже ти виводиш не тільки інформацію про площу, а загальну інформацію